docs/reference/commandline/volume_create.md

```diff
@@ -29,6 +29,8 @@ The mount is created inside the container's `/src` directory. Docker does not su
 
 Multiple containers can use the same volume in the same time period. This is useful if two containers need access to shared data. For example, if one container writes and the other reads the data.
 
+If a volume is already exist, you cannot create again with a different driver name and with the same volume name. If you create again with the same name and no drivername or the same dirvername, you just get the existing volume.
```

Comment:
`You cannot create a volume if there is a volume already registered with the same name in other driver. Docker returns the existent volume ID if the volume name conflicts and you don't specify the driver`